src/main/scala/com/twitter/finagle/postgres/PostgresClient.scala

```diff
@@ -69,7 +69,7 @@ object PostgresClient {
 
   case class TypeSpecifier(receiveFunction: String, typeName: String, elemOid: Long = 0)
 
-  private[finagle] val defaultTypes = Map(
+  val defaultTypes = Map(
```

Comment:
I'm making it public to make things slightly easier for the next person who finds that one of the types they use isn't part of `defaultTypes` and who doesn't want to use the fatally-bugged dynamic-type-map-building feature. Right now, if you want to use `withCustomTypes`, chances are that you'll have to copy this definition of `defaultTypes` into your code so you can have a `Map` that adds the mappings you need to pass to `withCustomTypes`.